backend/tessend/tessend.go

```diff
@@ -200,20 +217,44 @@ func (t *Tessend) startMessageHandler() {
 		}
 
 		tessen, ok := msg.(*corev2.TessenConfig)
-		if !ok {
-			logger.WithField("msg", msg).Errorf("received non-TessenConfig on tessen message channel")
-			return
+		if ok {
+			data := t.getDataPayload()
+			t.getTessenConfigMetrics(time.Now().UTC().Unix(), tessen, data)
```

Comment:
Unix time has no timezone; I'm not 100% sure how `UTC()` and `Unix()` interact in the standard library. I'm assuming that `UTC()` has no effect in this case. To be certain, I would just drop `UTC()` altogether!